<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caster Name List</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog {
	background-color: #1f232a;
}

#label, #label_2 {
	color: #FFFFFF
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>30</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_9"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit_7"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_10"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_11"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_12"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>PORT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portNumber">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>서버 위치</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="position">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>15</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="LocationList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="inputLocation"/>
        </item>
        <item>
         <widget class="QPushButton" name="addLocation">
          <property name="text">
           <string>입력</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delLocation">
          <property name="text">
           <string>삭제</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
